presentation/src/main/java/org/buffer/android/boilerplate/presentation/browse/model/ArticleView.kt

```diff
@@ -0,0 +1,12 @@
+package org.buffer.android.boilerplate.presentation.browse.model
+
+
+class ArticleView(
```

Comment:
Well this `presentation` module might be good to be combined with `mobile-ui` that is assigned to me haha